### PR TITLE
Add Sphinx 6.1.3 to test matrix

### PR DESCRIPTION
**Pull request by michaeljones**
But don't test it against 3.7 as it does not support that release.

### CODE REVIEW

**Comment by michaeljones** [comment]
CI seems to be happy.

**Comment by michaeljones** [comment]
@vermeeren - are you happy to merge this and do a release? I think we want to have something that accepts Sphinx 6. I've not tested it thoroughly but CI seems happy .

File: requirements/production.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ docutils>=0.12
 Jinja2>=2.7.3
 MarkupSafe>=0.23
 Pygments>=1.6
-Sphinx>=4.0,<6,!=5.0.0
+Sphinx>=4.0,<7,!=5.0.0
```
Comment (jakobandersen) [inline requirements/production.txt:5]:
```suggestion
Sphinx>=4.0,!=5.0.0
```
The release cadence of Sphinx has changed so major versions may now happen more often than once a year. I suggest we don't upper bound the version as we would have to be much more vigilant and update Breathe.

Comment (michaeljones) [reply requirements/production.txt:5]: Thanks @jakobandersen - should we do this in setup.py too? 

Cheers for reviewing it. I am very aware that you normally handle this but felt maybe you were busy. I should have @-ed you anyway :)

Comment (jakobandersen) [reply requirements/production.txt:5]: > should we do this in setup.py too?

Yes, I forgot about that having to be modified as well.

> Cheers for reviewing it. I am very aware that you normally handle this but felt maybe you were busy. I should have @-ed you anyway :)

No problem, I have indeed been too busy to handle it, so great you have done so :-).

**Comment by JasperCraeghs** [comment]
When will this be released? I'd like to start using Sphinx 6.1.3.

**Comment by michaeljones** [comment]
I have just released 4.35.0. I hope it is satisfactory.

**Comment by kartben** [comment]
Much appreciated, thank you!

**Pull request merged**